server/routes/api/graphql.ts

```diff
@@ -229,19 +272,37 @@ function userRecordToGraphql(user: IUser): types.User<Ctx> {
 
 	const confirmation: types.Branch<Ctx> | undefined = user.attending ? {
 		type: user.confirmationBranch,
-		data: user.confirmationData.map(recordToFormItem),
+		data: await Promise.all(user.confirmationData.map(item => recordToFormItem(item, user.confirmationBranch))),
 		start_time: user.confirmationStartTime &&
 			user.confirmationStartTime.toDateString(),
 		submit_time: user.confirmationSubmitTime &&
 			user.confirmationSubmitTime.toDateString()
 	} : undefined;
 
+	let loginMethods: string[] = [];
+	if (user.githubData && user.githubData.id) {
```

Comment:
Might want to refactor this later if we ever want to add other login providers in a modular way